### PR TITLE
T36916 Implement pagination for '/nodes' endpoint

### PR DESCRIPTION
**Pull request by JenySadadia**
Add 'skip' and 'limit' request query parameters to get a desired set of nodes in response from the endpoint.
Need to drop them from 'query_params' dictionary to match the fields with the model. It is required to get objects matching model attributes from the database.

### CODE REVIEW

**Review by gctucker — COMMENTED** [review]
How about reusing this?
https://pypi.org/project/fastapi-pagination/

**Comment by JenySadadia** [comment]
> How about reusing this? https://pypi.org/project/fastapi-pagination/

The reason being is FastAPI official documentation doesn't mention this package for pagination. 
Instead, it mentioned using query parameters to implement it the simpler way. https://fastapi.tiangolo.com/tutorial/query-params/
Also, I am unsure if the https://pypi.org/project/fastapi-pagination/ is thoroughly tested.

**Comment by gctucker** [comment]
> 

Looking at the GitHub page it runs over 500 unit tests on 4 Python revisions and also does pylint.  It's actively maintained so I think it's worth considering it.  Of course the FastAPI documentation doesn't mention external packages, but that doesn't mean they can't be used.

**Comment by JenySadadia** [comment]
> Looking at the GitHub page it runs over 500 unit tests on 4 Python revisions and also does pylint. It's actively maintained so I think it's worth considering it. Of course the FastAPI documentation doesn't mention external packages, but that doesn't mean they can't be used.

Okay. I think I'll give it a try.

File: api/models.py [hunk 1]
```diff
@@ -66,6 +67,18 @@ class Config:
...
+class PageModel(LimitOffsetPage[TypeVar("T")]):
+    """Model for pagination
+
+    This model is required to serialize paginated model data response"""
+
+    class Config:
+        """Configuration attributes for PageNode"""
+        json_encoders = {
+            ObjectId: str,
+        }
```
Comment (mgalka) [inline api/models.py:79]: I'd think of moving this model to different module. `models.py` seems to be a place for data models. These are connected with the application logic, while `PageModel` is rather connected with data presentation.

**Review by mgalka — COMMENTED** [review]
Small remarks

File: api/models.py (hunk 1 above)
Comment (JenySadadia) [reply api/models.py]: Makes sense. Added `paginator_models.py`

**Review by mgalka — APPROVED** [review]
LGTM

**Pull request merged**